modules/apps/headless/headless-object/headless-object-impl/src/main/java/com/liferay/headless/object/internal/resource/v1_0/ObjectEntryFolderResourceImpl.java

```diff
@@ -66,10 +67,16 @@
  */
 @Component(
 	properties = "OSGI-INF/liferay/rest/v1_0/object-entry-folder.properties",
+	property = {
+		"batch.engine.task.item.delegate.name=CMSFolder",
```

Comment:
Delegate names are not written in camel case (look for other instances of `batch.engine.task.item.delegate.name` for examples). Here, it should be `depot-object-entry-folder` or something similar.